classes/core/traits/ModelWithSettings.php

```diff
@@ -65,8 +65,13 @@ abstract protected function ensureCastsAreStringValues($casts);
     public function __construct(array $attributes = [])
     {
         parent::__construct($attributes);
+        
         if (static::getSchemaName()) {
             $this->setSchemaData();
+        } else {
+            if (!empty($this->fillable)) {
+                $this->mergeFillable(array_merge($this->getSettings(), $this->getMultilingualProps()));
+            }
```

Comment:
@Vitaliy-1 this changes seems required to populate the proper `fillable` as in the `if` condition, we are only considering entities with schema maps but also need to consider for entities with settings with no schema map .